website/docs/docs/build/ref-metrics-in-filters.md

```diff
@@ -0,0 +1,163 @@
+)
+
+select
+    *
+from 
+    activated_accounts
+```
+
+This previous SQL query calculates the number of `activated_accounts` by using the `data_model_runs` metric as a dimension for the user entity. It filters based on the metric value scoped to the account entity. You can express this logic natively in the MetricFlow specification.
+
+### Reference a metric in a filter
+
+You can recreate this pattern in MetricFlow by referencing a metric in the `where` filter for another metric using the `Metric()` object syntax. The function for referencing a metric takes a metric name and exactly one entity:
+
+```yaml
+{{ Metric('metric_name', group_by=['Entity']) }}
+```
+
+Using the same `activated_accounts` metric as an example:
+- Create two semantic models, `model_runs` and `accounts`. 
+- Then create a `measure` and `metric` to count data model runs, and another measure to count users.
+- Specify the foreign entity `account` in the `model_runs` semantic model.
+
+```yaml
+semantic_models:
+  - name: model_runs
+    .... # Placeholder for other configurations
+	entities:
+      - name: model_run
+        type: primary
+      - name: account
+        type: foreign
+    measures:
+      - name: data_model_runs
+        agg: sum
+        expr: 1
+        create_metric: true
+
+  - name: accounts
+  	.... # Placeholder for other configurations
+	entities:
+      - name: account
+        type: primary
+    measures:
+      - name: accounts
+        agg: sum
+        expr: 1
+```
+
+- Then create the `Activated Accounts` metric by filtering accounts that have more than five data model runs:
+
+```yaml
+metrics:
+ - name: activated_accounts
+	 label: Activated Accounts
+   type: simple
+   type_params:
+	   measure: accounts
+	 filter: |
+		 {{ Metric('data_model_runs', group_by=['account']) }} > 5
+```
+
+Let’s break down the SQL that will be generated from this metric definition when I run the following query from the CLI `dbt sl query --metrics activated_accounts` 
+
+First, the filter `{{ Metric('data_models', group_by=['account']) }}` will generate SQL similar to the `data_models_per_user` sub query shown above.
+
+```sql
+select
+  sum(1) as data_model_runs
+  , account
+ from data_model_runs
+ group by 
+	 account
+```
+
+We will then join this query to the query generated by the `accounts` measure on the group by elements and apply the filter conditions.
+
+```sql
+select
+	sum(1) as activated_accounts
+from accounts
+left join (
+select
+    sum(1) as data_model_runs
+  , account
+ from data_model_runs
+ group by 
+	 account
+) as subq
+on accounts.account = subq.account
+where data_model_runs > 5
+	
+```
+
+The intermediate tables we would use to create this metric are:
+
+Accounts with the `data_model_runs` dimension
+
+| account | data_model runs |
+| --- | --- |
+| 1 | 4 |
+| 2 | 7 |
+| 3 | 9 |
+| 4 | 1 |
+
+We’ll filter the above table for  accounts with >5 data model runs, then count the number of accounts who meet this criteria.
+
```

Comment:
Should we explicity say, "Then, using the `{{ Metric('data_model_runs', group_by=['account']) }}` filter, you canfilter this table to accounts with more than 5...."?

Reply:
Maybe we say basically that but referencing `MetricFlow` instead of `you` to maintain consistency with the rest of the section, since this is a continuation. i.e.,
"MetricFlow then filters this table to accounts with more than 5 data model runs."